segments/path.bash

```diff
@@ -2,6 +2,7 @@
 _sbp_path_color_bg=${_sbp_path_color_bg-:$_sbp_color_blue}
 _sbp_path_color_fg=${_sbp_path_color_fg-:$_sbp_color_white}
 _sbp_path_color_sep=${_sbp_path_color_sep-:$_sbp_color_grey}
+_sbp_path_disable_sep=${_sbp_path_disable_sep-:0}
```

Comment:
I couldn't find an explanation for the quotes in the posted link. The author always uses quotes for his examples, but doesn't give an explanation for this. However, I tested the code with and without quotes and both work fine.